test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -3035,12 +2984,12 @@ static public void testUnresolvedPutFieldUse() throws Throwable {
 	static public void testIdentityObjectOnValueType() throws Throwable {
 		String fields[] = {"longField:J"};
 		Class valueClass = ValueTypeGenerator.generateValueClass("testIdentityObjectOnValueType", fields);
-		assertFalse(Arrays.asList(valueClass.getInterfaces()).contains(IdentityInterface.class));
+		//assertFalse(Arrays.asList(valueClass.getInterfaces()).contains(IdentityInterface.class));
```

Comment:
you can leave these in for now, we will change the name in another PR

Reply:
keep the tests enabled, we will need a second pass at this. Next items will be to rename the IndentityInterface and add MH support for VTs. After we can try this again. For now the fixes you have are fine, just need to make sure the tests keep running.